packages/app/hooks/appearance/useAppearance.ts

```diff
@@ -0,0 +1,28 @@
+import useTheme from '../useTheme';
+import { useStorageState } from 'app/hooks/storage/useStorageState';
+
+const useAppearance = () => {
+  const { enableDarkMode, enableLightMode, currentTheme } = useTheme();
+  const [[, isEnabledString], setIsEnabled] = useStorageState('isEnabled');
+
+  // Convert string to boolean
+  const isEnabled = JSON.parse(isEnabledString);
+
+  const toggleSwitch = async () => {
+    const newIsEnabledValue = !isEnabled ? 'true' : 'false';
+    setIsEnabled(newIsEnabledValue);
+    if (!isEnabled) {
+      enableDarkMode();
+    } else {
+      enableLightMode();
+    }
+  };
+
+  return {
+    isEnabled,
```

Comment:
We don't use this hook anymore, we can delete it

Reply:
I am not sure if we should delete it, cause its meant to separate concern out of appearance page on toggle switch.